android/docs/engine/guides/in-app/index.md

```diff
@@ -0,0 +1,135 @@
+* Install [OpenJDK 11](https://learn.microsoft.com/en-us/java/openjdk/download#openjdk-11).
+* Install [Android Studio](https://developer.android.com/studio/index.html).
+    Android Studio runs on several platforms. Keyman Developer runs on Windows 10 or later.
+
+### 2. Configure Android Studio
+* For Windows users, from **Control Panel>System>System Properties>Environment Variables**,<br/>
+        create a system variable **ANDROID_HOME** to the location of your Android SDK. The default
+        installation location is<br/>
+        c:\Users\[USER]\AppData\Local\Android\sdk where [USER] is your Windows username.
+* For Linux users, add the following to ~/.bashrc
+```bash
+export ANDROID_HOME=$HOME/Android/Sdk
+export PATH=$PATH:$ANDROID_HOME/tools
+```
+* In a command prompt or terminal, accept all the SDK license agreements
+```bash
+cd c:\Users\[USER]\AppData\Local\Android\sdk\tools\bin
+yes | ./sdkmanager.bat --licenses
+```
+
+### 3. Download Keyman Engine for Android and Build Sample projects
+* Download the [Keyman for Android SDK](https://keyman.com/engine)
+        and extract the files to a new folder.
+
+The archive includes two sample projects and an Android .aar library file. This guide will use the first
+    example project, **KMSample1**.
+
+* If **KMSample1.zip** exists, extract it to a new folder. Otherwise, the KMSample1 project
+        can be found at **Samples/KMSample1**
+* In Android Studio, select **File&gt;Open** and choose the KMSample1 project folder from
+        the previous step.
+
+* When the project loads, you may be prompted to install Android SDKs; go ahead and follow the prompts to
+        fixup any missing SDK dependencies.
+
+### 4. Create a keyboard layout
+Use Keyman Developer to build a touch layout. The following blog posts walk through some of the development and
+    testing for creating a touch keyboard layout:
+
+* [Creating a Touch Keyboard Layout for Amharic with Keyman Developer 13](/developer/13.0/guides/develop/creating-a-touch-keyboard-layout-for-amharic-with-keyman-developer-10")
+* [Creating A Touch Keyboard Layout For Amharic — The Nitty Gritty](/developer/13.0/guides/develop/creating-a-touch-keyboard-layout-for-amharic-the-nitty-gritty)
+* [How to test your keyboard layout with Keyman Developer 13 — touch and desktop](/developer/13.0/guides/test/how-to-test-your-keyboard-layout-with-keyman-developer-10-touch-and-desktop)
+* [How to test your touch layout in the Google Chrome mobile emulator](/developer/13.0/guides/test/how-to-test-your-touch-layout-in-the-google-chrome-mobile-emulator)
+
+When your keyboard is ready, you should have a compiled keyboard package file. The example below shows the
+    Tamil 99 Basic touch layout.
+[<img src="/cdn/dev/img/engine/android/14.0/guides/in-app/touch-layout-800wi.png" title="Keyman-developer" alt="Keyman-developer">](/cdn/dev/img/engine/android/14.0/guides/in-app/touch-layout.png)
```

Comment:
I was trying to see how check-markdown validated external links and references to image assets.

I think I only saw the other links checked when I added the verbose flag (`-v`)

```bash
  node "$KEYMAN_ROOT/resources/tools/check-markdown" -v --root "$1"
```

This might get complicated cause the help.keyman.com assets are scattered in `/cdn/dev/img/android/`

Reply:
The check-markdown script explicitly ignores absolute and external references, for this reason. My thinking here is that over time we move the images into the engine folder tree and get rid of the /cdn/dev paths. Then that should resolve the problem of broken links there (and help us to keep content in sync with code versions as well). But that is for a follow-up PR, not for this initial one.